rust/email/src/lib.rs

```diff
@@ -36,20 +39,35 @@ fn c_char_to_string(str_: *const c_char) -> Result<String, i32> {
 	}
 }
 
+/// Reads the email body from a file, located in `filepath`.
+///
+/// # Arguments
+///
+/// * `filepath` - A String containing the filepath of the file
+///   we want to read.
+fn read_mail_body(filepath: &str) -> Result<String, io::Error> {
+	let body = fs::read_to_string(filepath);
+
+	return body;
+}
```

Comment:
Let's be more clear, like,
```suggestion
fn read_mail_body(filepath: &str) -> Result<String, io::Error> {
	 fs::read_to_string(filepath)
}
```